spark-extension/src/main/java/org/apache/spark/sql/auron/AuronConf.java

```diff
@@ -67,64 +68,138 @@ public enum AuronConf {
 
-    // spark io compression codec
+    /// spark io compression codec
     SPARK_IO_COMPRESSION_CODEC("spark.io.compression.codec", "lz4"),
 
-    // spark io compression zstd level
+    /// spark io compression zstd level
     SPARK_IO_COMPRESSION_ZSTD_LEVEL("spark.io.compression.zstd.level", 1),
 
-    // tokio worker threads per cpu (spark.task.cpus), 0 for auto detection
+    /// tokio worker threads per cpu (spark.task.cpus), 0 for auto-detection
     TOKIO_WORKER_THREADS_PER_CPU("spark.auron.tokio.worker.threads.per.cpu", 0),
 
-    // number of cpus per task
+    /// number of cpus per task
     SPARK_TASK_CPUS("spark.task.cpus", 1),
 
-    // replace all sort-merge join to shuffled-hash join, only used for benchmarking
+    /// replace all sort-merge join to shuffled-hash join, only used for benchmarking
     FORCE_SHUFFLED_HASH_JOIN("spark.auron.forceShuffledHashJoin", false),
 
-    // shuffle compression target buffer size, default is 4MB
+    /// shuffle compression target buffer size, default is 4MB
     SHUFFLE_COMPRESSION_TARGET_BUF_SIZE("spark.auron.shuffle.compression.targetBufSize", 4194304),
 
-    // spark spill compression codec
+    /// spark spill compression codec
     SPILL_COMPRESSION_CODEC("spark.auron.spill.compression.codec", "lz4"),
 
-    // enable hash join falling back to sort merge join when hash table is too big
+    /// enable hash join falling back to sort merge join when hash table is too big
     SMJ_FALLBACK_ENABLE("spark.auron.smjfallback.enable", false),
 
-    // smj fallback threshold
+    /// smj fallback threshold
     SMJ_FALLBACK_ROWS_THRESHOLD("spark.auron.smjfallback.rows.threshold", 10000000),
 
-    // smj fallback threshold
+    /// smj fallback threshold
     SMJ_FALLBACK_MEM_SIZE_THRESHOLD("spark.auron.smjfallback.mem.threshold", 134217728),
 
-    // max memory fraction of on-heap spills
+    /// max memory fraction of on-heap spills
     ON_HEAP_SPILL_MEM_FRACTION("spark.auron.onHeapSpill.memoryFraction", 0.9),
 
-    // suggested memory size for record batch
+    /// suggested memory size for record batch
     SUGGESTED_BATCH_MEM_SIZE("spark.auron.suggested.batch.memSize", 8388608),
 
-    // fallback to UDFJson when error parsing json in native implementation
+    /// fallback to UDFJson when error parsing json in native implementation
     PARSE_JSON_ERROR_FALLBACK("spark.auron.parseJsonError.fallback", true),
 
-    // suggested memory size for k-way merging
-    // use smaller batch memory size for kway merging since there will be multiple
-    // batches in memory at the same time
+    /// suggested memory size for k-way merging
+    /// use smaller batch memory size for kway merging since there will be multiple
+    /// batches in memory at the same time
     SUGGESTED_BATCH_MEM_SIZE_KWAY_MERGE("spark.auron.suggested.batch.memSize.multiwayMerging", 1048576),
 
+    /// enable scan operation
+    ENABLE_SCAN("spark.auron.enable.scan", true),
```

Comment:
The `spark.auron.enabled` configuration item can also be moved to `AuronConf`.

https://github.com/kwai/auron/blob/300592d1b0bfdabb14012f33231ab929a6d11bfb/spark-extension/src/main/scala/org/apache/spark/sql/auron/AuronSparkSessionExtension.scala#L48-L51